frontend/src/common/enums/notification/notification-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum NotificationMessage {
```

Comment:
good, like it 👍
could you apply these enums for all notifications?

Reply:
yep